docs/src/firmware-action/interactive.md

```diff
@@ -0,0 +1,36 @@
+# Interactive mode
```

Comment:
Should we already but a note on this page, that since dagger 0.12 it is also supported upstream?

Reply:
We could add a note and also link to issue #269 where we plan to re-write `firmware-action` to use this native dagger feature.